Orange/widgets/visualize/owheatmap.py

```diff
@@ -565,18 +524,13 @@ def __init__(self):
                  label="Clusters:", keyboardTracking=False,
                  callbackOnReturn=True, callback=self.update_merge)
 
-        sortbox = gui.vBox(self.controlArea, "Sorting")
-        # For columns
-        self.colsortcb = gui.comboBox(
-            sortbox, self, "sort_columns_idx",
-            items=[name for _, name in self.ColumnOrdering],
-            label='Columns', callback=self.update_sorting_attributes)
-
-        # For rows
-        self.rowsortcb = gui.comboBox(
-            sortbox, self, "sort_rows_idx",
-            items=[name for _, name in self.RowOrdering],
-            label='Rows', callback=self.update_sorting_examples)
+        cluster_box = gui.vBox(self.controlArea, "Clustering")
```

Comment:
I would rename this to Cluster (as the section above has title Merge, not Merging).